<ui version="4.0" >
 <class>HelpWidget</class>
 <widget class="QMainWindow" name="HelpWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>798</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>OpenFlipper Help</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QWidget" name="gridLayoutWidget" >
    <property name="geometry" >
     <rect>
      <x>9</x>
      <y>40</y>
      <width>951</width>
      <height>731</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_" />
   </widget>
   <widget class="QToolButton" name="backButton_" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximumSize" >
     <size>
      <width>16777215</width>
      <height>31</height>
     </size>
    </property>
    <property name="toolTip" >
     <string>Go one page back.</string>
    </property>
    <property name="whatsThis" >
     <string>Go one page back.</string>
    </property>
    <property name="text" >
     <string>Back</string>
    </property>
   </widget>
   <widget class="QToolButton" name="forwardButton_" >
    <property name="geometry" >
     <rect>
      <x>70</x>
      <y>10</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximumSize" >
     <size>
      <width>16777215</width>
      <height>31</height>
     </size>
    </property>
    <property name="toolTip" >
     <string>Go one page forward.</string>
    </property>
    <property name="whatsThis" >
     <string>Go one page forward.</string>
    </property>
    <property name="text" >
     <string>Forward</string>
    </property>
   </widget>
   <widget class="QToolButton" name="homeButton_" >
    <property name="geometry" >
     <rect>
      <x>160</x>
      <y>10</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximumSize" >
     <size>
      <width>16777215</width>
      <height>31</height>
     </size>
    </property>
    <property name="toolTip" >
     <string>Open home site.</string>
    </property>
    <property name="whatsThis" >
     <string>Open home site.</string>
    </property>
    <property name="text" >
     <string>Home</string>
    </property>
   </widget>
   <widget class="QToolButton" name="searchButton_" >
    <property name="geometry" >
     <rect>
      <x>910</x>
      <y>10</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximumSize" >
     <size>
      <width>16777215</width>
      <height>31</height>
     </size>
    </property>
    <property name="toolTip" >
     <string>Search help for specified item.</string>
    </property>
    <property name="whatsThis" >
     <string>Search help for specified item.</string>
    </property>
    <property name="text" >
     <string>Search</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
 </widget>
 <resources/>
 <connections/>
</ui>
